maintainers/maintainer-list.nix

```diff
@@ -8821,4 +8821,10 @@
     github = "cpcloud";
     githubId = 417981;
   };
+  superbo = {
```

Comment:
The addition to `maintainer-list.nix` should be in a separate commit

Reply:
@alexarice : thank you, I just reformat my commits.